css/style.css

```
@@ -0,0 +1,182 @@
+:root {
+    --basic-black: #000000;
+    --basic-white: #ffffff;
+    --basic-blue: #23A9E1;
+    --basic-grey: #333333;
+    --basic-brown: #766357;
+    --basic-blue1: #81B3D2;  
+    --basic-white1: #F2F2F2;
+    --basic-grey2: #A9A9A9;
+    --basic-blue2: #81B3D2;
+    --basic-white2: #eeeeee;
+}
+
+.page-body {
+  min-width: 960px;
+  margin: 0;
+  padding: 0;
+  font-family: "PT Sans", Arial, sans-serif;
+  font-size: 21px;
+  line-height: 26px;
+  font-weight: 700px;
```

1. В корневом элементе мы задаём стили шрифта, который чаще всего используется на сайте. Это текст статей. `font-weight: 700` используется только в некоторых случаях, а основное начертание текста на сайте - 400.
2. `font-weight` не измеряется в пикселах. Должно быть:
```suggestion
  font-weight: 400;
```